tools/travis/flake8.sh

```diff
@@ -14,7 +14,7 @@ declare -a PYTHON_FILES=("."
 echo 'Flake8: first round (fast fail) stops the build if there are any Python 3 syntax errors...'
 for i in "${PYTHON_FILES[@]}"
 do
-    flake8 "$i" --select=E999 --statistics
+    flake8 "$i" --select=E999,F821 --statistics
```

Comment:
Should we rather do this the other way round: Fail on everything and only exclude the rules we don't like?